### ISSUE TITLE
Getting error when using DriverInstaller to install .sys driver

### ISSUE THREAD

**Issue by renishnair-viper** [oleg-shilo/wixsharp#1338]
Hello Team,

We are using WixSharp to create an installer (setup) for our WPF application. There is one requirement to add (kernel mode) driver within the installation process.  So there is an .sys file for 32bit or 64bit which will be used. We are using this example 
https://github.com/oleg-shilo/wixsharp/blob/7601893a5e9e14408ce9ef63415290d5c56254a6/Source/src/WixSharp.Samples/Wix%23%20Samples/Driver/setup.cs
to add a driver file but somehow we are getting errors.
```C#
var project = new ManagedProject(application_name,new Dir(new Id("Drivers"),
                new WixSharp.File(driverFile,
                                            new DriverInstaller
                                            {
                                                ForceInstall = true,
                                                AddRemovePrograms = true,
                                                DeleteFiles = true,
                                                Legacy = true,
                                                PlugAndPlayPrompt = false,
                                                Sequence = 1,
                                                Architecture = (platform == Platform.x64) ? DriverArchitecture.x64 : DriverArchitecture.x86
                                            })
                ));
```
On installation, all files are properly copied to the installation location but when executed internally get the below error:
Error 0x80070643: Failed to install MSI package.
Error 0x80070643: Failed to execute MSI package.
Error 0x80070643: Failed to configure per-machine MSI package.
Applied execute package: MainPackage_x64, result: 0x80070643, restart: None
Failed to execute MSI package.

Thanks,
Renish

**Comment by oleg-shilo** [comment]
Unfortunately, MSI runtime is notoriously bad at error logging. MSI/WiX is failing to install the driver for some reason. It can be related to permissions, CPU architecture or a driver failure. Who knows. :o(

I can only suggest you have a look at the generated wxs file in case it can give you some clues.

Check system events....

You can also try to verify that you can install your driver manually with `sc.exe`. If it works then you can see if you can repeat the same installation with `sc.exe` but from the `AfterInstall` event handler in your installer. Just to see that the MSI process has the required elevation.

Basically try to digg as much as possible.
